Framework/DataHandling/src/LoadBankFromDiskTask.cpp

```diff
@@ -470,24 +468,16 @@ void LoadBankFromDiskTask::run() {
   auto numEvents = static_cast<size_t>(m_loadSize[0]);
   auto startAt = static_cast<size_t>(m_loadStart[0]);
 
-  // convert things to shared_arrays to share between tasks
```

Comment:
This comment indicates why we need to be careful with moving this to vectors. The arrays are shared between the `ProcessBankData` tasks that are run on different threads and moving them to vectors means they are going to be copied and reduce the performance of this.

I think a shared_ptr<vector> would work?

Reply:
As per the comment, these are unique pointers to an array which get converted to shared arrays however we can't use shared arrays due to lack of compiler support. Is it still okay to just use a shared_ptr?